docs/source/process.rst

```diff
@@ -22,75 +21,40 @@ Functionality
 
 - EK80 and EA640 broadband echosounders:
 
-  - calibration based on pulse compression output in the
+  - Calibration based on pulse compression output in the
     form of average over frequency.
 
+  - The same noise removal and MVBS functionality available
+    to the narrowband echosounders.
 
-The steps of performing these analysis for EK60 and AZFP echosounders
-are summarized below.
-Additional information will be added for broadband EK80 and EA640 echosounders as
-additional functionality is developed.
 
-.. code-block:: python
-
-   from echopype import Process
-   nc_path = './converted_files/convertedfile.nc'  # path to a converted nc file
-   ed = Process(nc_path)    # create a processing object
-   ed.calibrate()           # Sv
-   ed.remove_noise()        # denoise
-   ed.get_MVBS()            # calculate MVBS
-
-By default, these methods do not save the calculation results to disk.
-The computation results can be accessed from ``ed.Sv``, ``ed.Sv_clean`` and
-``ed.MVBS`` as xarray Datasets with proper dimension labels.
-
-To save results to disk:
+The steps for performing these analyses are summarized below.
 
 .. code-block:: python
 
-   ed.calibrate(save=True)     # output: convertedfile_Sv.nc
-   ed.remove_noise(save=True)  # output: convertedfile_Sv_clean.nc
-   ed.get_MVBS(save=True)      # output: convertedfile_MVBS.nc
-
-
-There are various options to save the results:
-
-.. code-block:: python
-
-   # Overwrite the output postfix from _Sv to_Cal: convertedfile_Cal.nc
-   ed.calibrate(save=True, save_postfix='_Cal')
-
-   # Save output to another directory: ./cal_results/convertedfile_Sv.nc
-   ed.calibrate(save=True, save_path='./cal_results')
-
-   # Save output to another directory with an arbitrary name
-   ed.calibrate(save=True, save_path='./cal_results/somethingnew.nc')
+   import echopype as ep
+   nc_path = './converted_files/file.nc'           # path to a converted nc file
+   echodata = ep.open_converted(nc_path)           # create a processing object
```

Comment:
> create a processing object

That's no longer a `processing` object, per se. Change to "create an EchoData object"